_licenses/postgresql.txt

```diff
@@ -0,0 +1,50 @@
+---
+title: PostgreSQL License
+spdx-id: PostgreSQL
+source: https://opensource.org/licenses/PostgreSQL
+featured: false
+hidden: true
+
+description: A very short, BSD-style license, used specifically for PostgreSQL.  
+
+how: To use it, say that it is The PostgreSQL License, and then substitute the copyright year and name of the copyright holder into the body of the license. Then put the license into a prominent file ("COPYRIGHT", "LICENSE" or "COPYING" are common names for this file) in your software distribution.
+
+using:
+  - PostgreSQL: https://www.postgresql.org/about/licence/
+  - PGAudit: https://github.com/pgaudit/pgaudit/blob/master/LICENSE
+  - .NET Access to PostgreSQL: https://github.com/npgsql/npgsql/blob/dev/LICENSE.txt
+
+permissions:
+  - commercial-use
+  - modifications
+  - distribution
+  - private-use
+
+conditions:
+  - include-copyright
+
+limitations:
+  - liability
+  - warranty
+
+---
+
+The PostgreSQL Licence (PostgreSQL)
```

Comment:
Maybe this should just be "PostgreSQL License"?

The headings on license pages at opensource.org aren't super reliable indicators of what should be included in the license text. Some related discussion for similar licenses at https://github.com/github/choosealicense.com/pull/371